integration_tests/test_model_upload.py

```diff
@@ -0,0 +1,31 @@
+import os
+import tempfile
+import unittest
+
+from kagglehub import model_upload, models_helpers
+
+LICENSE_NAME = "MIT"
+OWNER_SLUG = "integrationtester"
```

Comment:
For the creation case, let's create the model under the username of the person creating the test.

Reply:
Using from kagglehub.config import get_kaggle_credentials